### ISSUE TITLE
Wrong line separators error

### ISSUE THREAD

**Issue by whyleee** [jimkyndemeyer/js-graphql-intellij-plugin#347]
**Describe the bug**
After some recent extension update we started getting "Wrong line separators: '...roperties.\r\nUsually ...'" error. Seems like a regression of #191, maybe introduced by #341?

**To Reproduce**
1. Create GraphQL schema with field comments containing Windows line separators
2. Add ".graphqlconfig" referencing the endpoint exposing this schema
3. Click on "Get GraphQL Schema from Endpoint" action in GraphQL extension tab
4. "schema.graphql" file is not updated, I get this exception:

```
java.lang.AssertionError: Wrong line separators: '...roperties.\r\nUsually ...' at offset 5031
	at com.intellij.openapi.util.text.StringUtil.assertValidSeparators(StringUtil.java:2811)
	at com.intellij.openapi.editor.impl.DocumentImpl.assertValidSeparators(DocumentImpl.java:689)
	at com.intellij.openapi.editor.impl.DocumentImpl.replaceString(DocumentImpl.java:605)
	at com.intellij.openapi.editor.impl.DocumentImpl.lambda$setText$2(DocumentImpl.java:1060)
	at com.intellij.openapi.command.impl.CoreCommandProcessor.executeCommand(CoreCommandProcessor.java:220)
	at com.intellij.openapi.command.impl.CoreCommandProcessor.executeCommand(CoreCommandProcessor.java:178)
	at com.intellij.openapi.command.impl.CoreCommandProcessor.executeCommand(CoreCommandProcessor.java:168)
	at com.intellij.openapi.command.impl.CoreCommandProcessor.executeCommand(CoreCommandProcessor.java:154)
	at com.intellij.openapi.editor.impl.DocumentImpl.setText(DocumentImpl.java:1065)
	at com.intellij.lang.jsgraphql.ide.editor.GraphQLIntrospectionHelper.setEditorTextAndFormatLines(GraphQLIntrospectionHelper.java:369)
	at com.intellij.lang.jsgraphql.ide.editor.GraphQLIntrospectionHelper.lambda$createOrUpdateIntrospectionOutputFile$9(GraphQLIntrospectionHelper.java:356)
	at com.intellij.openapi.application.impl.ApplicationImpl.runWriteAction(ApplicationImpl.java:976)
	at com.intellij.lang.jsgraphql.ide.editor.GraphQLIntrospectionHelper.createOrUpdateIntrospectionOutputFile(GraphQLIntrospectionHelper.java:332)
	at com.intellij.lang.jsgraphql.ide.editor.GraphQLIntrospectionHelper$2.lambda$run$0(GraphQLIntrospectionHelper.java:152)
	at com.intellij.openapi.application.TransactionGuardImpl$2.run(TransactionGuardImpl.java:201)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:831)
	at com.intellij.openapi.application.impl.ApplicationImpl.lambda$invokeLater$4(ApplicationImpl.java:310)
	at com.intellij.openapi.application.impl.FlushQueue.doRun(FlushQueue.java:80)
	at com.intellij.openapi.application.impl.FlushQueue.runNextEvent(FlushQueue.java:128)
	at com.intellij.openapi.application.impl.FlushQueue.flushNow(FlushQueue.java:46)
	at com.intellij.openapi.application.impl.FlushQueue$FlushNow.run(FlushQueue.java:184)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:313)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:776)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:727)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:746)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:974)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:847)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$8(IdeEventQueue.java:449)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:741)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$9(IdeEventQueue.java:448)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:831)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:496)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
```

**Expected behavior**
Schema is properly updated (same GraphQL endpoing was working earlier this year)

**Version and Environment Details**
Operation system: Windows 10
IDE name and version: Rider 2020.1.3
Plugin version: 2.5.0

**Comment by jimkyndemeyer** [comment]
Hi Pavel,

Thanks for using the plugin.

Can you try 2.4.0 from https://plugins.jetbrains.com/plugin/8097-js-graphql/versions to confirm whether this is regression in 2.5.0?

You can temporarily install 2.4.0 it using the gear icon in the plugins dialog and loading it from disk.

Best regards,
Jim.

**Comment by whyleee** [comment]
Hi Jim.

Yes, 2.4.0 works for me, thanks for the suggestion - will use that version for now 🙂 

So the issue introduced in 2.5.0.

And awesome plugin, thanks for the great work you do! 👍 👍

**Comment by malbanese** [comment]
I can confirm that I'm seeing this too.

**Comment by vepanimas** [comment]
Fixed in #354.
